docs/COMPONENT_GUIDELINES.md

```diff
@@ -29,3 +29,80 @@ So solutions like `@media-queries` might not work, and `window` shouldn't be dir
 
 * `wix-ui-tpa` are rendered for Wix's users-of-users, and so they must comply with web accessibility
  principles.
+ 
+ ## Typography
+ When creating a new component that renders text, we should in most cases use the `<Text/>` component with the correct props and style.  
+ In some cases we might need to use a typography that isn't defined in the `<Text/>` component.  
+ In this case, we rely on the style processor's syntax, to ensure we get the correct font from the viewer.
+ 
+ 
+ ### How it works
+ We use the [style processor's syntax](https://github.com/wix/wix-style-processor) to get our components wired correctly:  
+ The platform (viewer/editor) references the different fonts by a theme name, such as `Body-M` for paragraph text, and `Heading-L` for titles.  
+ To use these fonts we write for instance:
+ ```css
+:import {
+  -st-from: "src/common/formatters.st.js";
```

Comment:
This is internally in the library. how do we expect consumers to use it? is it even exported?

Reply:
This is part of the contribution guide, and not a consumer guide.
Formatters are exported, but this is an example for internal use